src/Columns/ColumnUnique.h

```diff
@@ -298,6 +300,51 @@ size_t ColumnUnique<ColumnType>::getNullValueIndex() const
     return 0;
 }
 
+template <typename ColumnType>
+void ColumnUnique<ColumnType>::insertWithGetTransIndex(const IColumn & src, size_t start, size_t length,
+                                                        std::unordered_map<UInt64, UInt64>& trans)
```

Comment:
Looks like we can use `PODArray<UInt64>` instead of `std::unordered_map` here (or maybe just `UInt64 *`)

Reply:
Also, it is not clear what is the difference with `uniqueInsertRangeFrom` ...

Reply:
Here, we do the dictionary merge, the src dictionary from gathered parts, during the merge I need get the diff index from the result dictionary index. So the trans table used to align the index from gathered parts to the result gathered column. You can check ColumnLowCardinality::Index::transformIndex. Maybe other type (like PODArray<UInt64> or UInt64 *) can not do that efficiently.  Thanks.